models/linkedin_ads__account_report.sql

```diff
@@ -0,0 +1,50 @@
+with account as (
+
+    select *
+    from {{ var('account_history') }}
+    where valid_to is null -- get latest 
+),
+
+campaign as (
+
+    select *
+    from {{ var('campaign_history') }}
+    where valid_to is null
+),
+
+report as (
+
+    select *
+    from {{ var('ad_analytics_by_campaign') }}
+),
+
+final as (
+
+    select 
+        report.date_day,
+        account.account_id,
+        account.account_name,
+        account.version_tag,
+        account.currency,
+        account.status,
+        account.type,
+        account.last_modified_at,
+        account.created_at,
+        sum(report.clicks) as clicks,
+        sum(report.impressions) as impressions,
+        sum(report.cost) as cost
+
+        {{ fivetran_utils.persist_pass_through_columns('linkedin_ads__passthrough_metrics', transform='sum') }}
```

Comment:
Same comment from the source regarding our decision to possibly split this out into two variables.

Reply:
The change suggested here would need to be applied to the other models if we decide to update.

Reply:
split the variables